ironfish-rust/src/frost_utils/split_secret.rs

```diff
@@ -21,7 +23,7 @@ pub fn split_secret(
     config: &SecretShareConfig,
     identifiers: IdentifierList,
     rng: &mut ThreadRng,
-) -> Result<(HashMap<Identifier, KeyPackage>, PublicKeyPackage), Error> {
+) -> Result<(HashMap<Identifier, KeyPackage>, PublicKeyPackage), IronfishError> {
     let secret_key = SigningKey::deserialize(
         config
             .secret
```

Comment:
Down on line 50 I missed an `unwrap`, that should be a `.map_err` to an `IronfishError`. Can you make that change too?

Reply:
Line 32 you should also convert to an `IronfishError`

Reply:
Also line 44

Reply:
Done